src/components/asset-item.tsx

```diff
@@ -117,7 +119,7 @@ export const AssetItem = memo(
                     label={formatted.isAbbreviated ? amount : undefined}
                   >
                     <Text fontVariantNumeric="tabular-nums" textAlign="right">
-                      {formatted.value}
+                      <span data-testid={name}>{formatted.value}</span>
```

Comment:
Here we could just add `data-testid` to the text above, rather than creating a new span

Reply:
I want to come back to this I pulled this out into a separate span because I wanted to do a bit more to the test. Since this is the first pass I stopped here but will add to this in a later pull request